app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
We can fix the locale files for that in a tiny PR after we get all of the sections completed with the PivT feature https://www.pivotaltracker.com/story/show/133316647

I'd really like to get the feature done -- touching all of the models, controllers, features, locale files.   :-)